lib/workload/stateless/stacks/metadata-manager/app/management/commands/insert_mock_data.py

```diff
@@ -1,22 +1,166 @@
 
 class Command(BaseCommand):
+    """
+    python manage.py insert_mock_data
+    """
     help = "Generate mock Metadata into database for local development and testing"
 
     def handle(self, *args, **options):
-        print("insert data from proc service test")
+        clear_all_data()
+        load_mock_from_wfm()
 
-        mock_sheet_data = [RECORD_1, RECORD_2, RECORD_3]
 
-        metadata_pd = pd.json_normalize(mock_sheet_data)
-        metadata_pd = sanitize_lab_metadata_df(metadata_pd)
-        result = persist_lab_metadata(metadata_pd, SHEET_YEAR)
+def load_mock_from_proc():
+    """Not in use for now, as loading data from wfm is preferred to sync data"""
+    mock_sheet_data = [RECORD_1, RECORD_2, RECORD_3]
 
-        print(json.dumps(result, indent=4))
-        print("insert mock data completed")
+    metadata_pd = pd.json_normalize(mock_sheet_data)
+    metadata_pd = sanitize_lab_metadata_df(metadata_pd)
+    result = persist_lab_metadata(metadata_pd, SHEET_YEAR, is_emit_eb_events=False)
+
+    print(json.dumps(result, indent=4))
+    print("insert mock data completed")
+
+
+def load_mock_from_wfm():
+    # The libraries are taken from WFM as of 16/10/2024
+    # Will sync this so test data sync across MM <=> WFM
+    libraries = [
+        {
+            "orcabus_id": "01J5M2JFE1JPYV62RYQEG99CP1",
+            "phenotype": "tumor",
+            "library_id": "L000001",
+            "assay": "TsqNano",
+            "type": "WGS",
+            "subject": "SBJ00001",
+            "workflow": "clinical"
+        },
+        {
+            "orcabus_id": "02J5M2JFE1JPYV62RYQEG99CP2",
+            "phenotype": "normal",
+            "library_id": "L000002",
+            "assay": "TsqNano",
+            "type": "WGS",
+            "subject": "SBJ00001",
+            "workflow": "clinical"
+        },
+        {
+            "orcabus_id": "03J5M2JFE1JPYV62RYQEG99CP3",
+            "phenotype": "tumor",
+            "library_id": "L000003",
+            "assay": "TsqNano",
+            "type": "WGS",
+            "subject": "SBJ00002",
+            "workflow": "research"
+        },
+        {
+            "orcabus_id": "04J5M2JFE1JPYV62RYQEG99CP4",
+            "phenotype": "normal",
+            "library_id": "L000004",
+            "assay": "TsqNano",
+            "type": "WGS",
+            "subject": "SBJ00002",
+            "workflow": "research"
+        },
+        {
+            "orcabus_id": "05J5M2JFE1JPYV62RYQEG99CP5",
+            "phenotype": "tumor",
+            "library_id": "L000005",
+            "assay": "ctTSOv2",
+            "type": "ctDNA",
+            "subject": "SBJ00003",
+            "workflow": "clinical"
+        },
+        {
+            "orcabus_id": "06J5M2JFE1JPYV62RYQEG99CP6",
+            "phenotype": "tumor",
+            "library_id": "L000006",
+            "assay": "ctTSOv2",
+            "type": "ctDNA",
+            "subject": "SBJ00003",
+            "workflow": "research"
+        },
+    ]
```

Comment:
Just dropping a note. No action need for the comment.

We should reorganise this as centralised seed data (in JSON or YAML format) somewhere in `docs` or `shared` directory down the track. It will get complex; when we include scenario and more services in the role play -- Case Manager, etc.

For current O3 iteration, this is fine.